compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -2612,7 +2629,7 @@ protected ParserRuleContext getNextRule(ParserRuleContext currentCtx, int nextLo
                         grandParentCtx == ParserRuleContext.OBJECT_MEMBER_DESCRIPTOR) {
                     return ParserRuleContext.OPTIONAL_RELATIVE_PATH;
                 }
-                return ParserRuleContext.OPEN_PARENTHESIS;
+                return ParserRuleContext.FUNC_NAME_RHS;
```

Comment:
With https://github.com/ballerina-platform/ballerina-lang/pull/28233#discussion_r579982420, we can revert this change.

Reply:
Fixed.